Testing/SystemTests/lib/systemtests/stresstesting.py

```diff
@@ -563,23 +577,28 @@ def start(self, script):
 #########################################################################
 class TestScript(object):
 
-    def __init__(self, test_dir, module_name, test_cls_name):
+    def __init__(self, test_dir, module_name, test_cls_name, exinpr):
         self._test_dir = test_dir
         self._modname = module_name
         self._test_cls_name = test_cls_name
+        self._exinpr = not exinpr
 
     def asString(self):
         code = '''import sys
 sys.path.append('{0}')
 sys.path.append('{1}')
 from {2} import {3}
 systest = {3}()
+def returnFalse():
```

Comment:
This is a great case for using a [lambda function](http://www.diveintopython.net/power_of_introspection/lambda_functions.html).
```python
if {5}:
    systest.excludeInPullRequests = lambda : False
```